kpops/components/common/topic.py

```diff
@@ -60,7 +60,10 @@ class TopicConfig(DescConfigModel):
     configs: dict[str, str | int] = Field(
         default={}, description=describe_attr("configs", __doc__)
     )
+    # TODO: We can rename this but this would be a breaking change
```

Comment:
Please rename. Breaking change is fine, we just need to be able to deploy streams-bootstrap v2 and v3 within the same project

Reply:
I will do this in a separate PR that merges into v8. If I change this all the SNAPSHOTs are going to change and this PR becomes even bigger